domains/cat.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Dogeyisyes",
+           "email": "dogegamesroblox@gmail.com",
+           "discord": "1128124810619273326"
+        },
+    
+        "record": {
+            "CNAME": "https://github.com/dogeyisyes/asdf3/blob/main/yes"
```

Comment:
Invalid CNAME record. CNAME records must be single string hostname's without any protocols or path URLs attached.

Reply:
Also this page returns a 404.